README.md

```diff
@@ -409,6 +531,182 @@ Client.event({
 })
 ```
 
+* **Update an event** - ```updateEvent```.
+This API updates an event. Find below the list of parameters supported by the API:
+
+```JavaScript
+event_id (Mandatory): string,
+name (Mandatory): string,
+response (Mandatory): Record<string, any>,
+```
+
+```JavaScript
+try{
+    const data = await Client.updateEvent({
+        event_id:"<Event Id of the event>",
+        name: "Event 1",
+        response: {
+            "<Key 1>": "<Value 1, Type: Any>",
+            "<Key 2>": "<Value 2, Type: Any>",
+            ...
+        }
+    })
+}catch(error){
+    //Catch any error
+}
+```
+
+OR
+
+```JavaScript
+Client.updateEvent({
+    event_id:"<Event Id of the event>",
+    name: "Event 1",
+    response: {
+        "<Key 1>": "<Value 1, Type: Any>",
+        "<Key 2>": "<Value 2, Type: Any>",
+        ...
+    }
+}).then(data=>{
+    console.log("data", data)
+}).catch(e=>{
+    console.log("error", e.message)
+})
+```
+
+* **Delete an event** - ```deleteEvent```.
+This API deletes an event. Find below the list of parameters supported by the API:
+
+```JavaScript
+event_id (Mandatory): string
+```
+
+```JavaScript
+try{
+    const data = await Client.deleteEvent({
+        event_id:"<Event Id of the event>"
+    })
+}catch(error){
+    //Catch any error
+}
+```
+
+OR
+
+```JavaScript
+Client.deleteEvent({
+    event_id:"<Event Id of the event>"
+}).then(data=>{
+    console.log("data", data)
+}).catch(e=>{
+    console.log("error", e.message)
+})
+```
+
+* **Trigger an event** - ```triggerEvent```.
```

Comment:
Lets not mention CRUD on events, in backend sdk.

`triggerEvent` and `event` do the same thing